templates/dashboard.jinja.html

```diff
@@ -446,13 +446,20 @@ <h1>
         {% for stage in pipe["ci_stages"] %}
         <div class="pipeline-stage">
 
-          <a class="stage-artifacts-link {{ stage['status'] }}"
-             href="{{ stage['url'] }}/index.html">
+          {% if stage['jobs'] %}
```

Comment:
Just wrap the `<a>` and `</a>` in `{% if %}`, no need for `elif`. This implementation duplicates the `stage-bar` div

Reply:
That doesn't seem right... I just tried it now, it looks fine:
![Screenshot 2022-07-12 at 19 10 14](https://user-images.githubusercontent.com/2887605/178563834-9bb8eefc-b6d9-4d8a-80f5-ed6830800af2.png)

I think you deleted the `<a>` tag but also everything inside it?

As you see in the inspector, I just deleted the `<a>` that is inside the `pipeline-stage` and that surrounds the `stage-bar`, but I kept `stage-bar`, that's what I'm proposing to do here. So you'd wrap the `<a>` and `</a>` in `{% if %}` but keep the stuff that is currently inside `<a>`... sorry that I wasn't clear the first time.